vast.conf

```
@@ -1,116 +1,380 @@
+; This is an example configuration file for VAST, striving to show all available
+; options. Lines starting with a semicolon are commented out. Options in angle
+; brackets have their default value determined at runtime.
+
+; Options that apply to VAST.
 system {
-;; The host and port to listen at and connect to.
-; endpoint = "localhost:42000"
+  ; The host and port to listen at and connect to.
+  ;endpoint = "localhost:42000"
+
+  ; The file system path used for persistent state.
+  ;db-directory = "vast.db/"
+
+  ; The file system path used for log files.
+  ;log-file = <server.log in db-directory>
+
+  ; Number of events to be packaged in a table slice (this is a target value that
+  ; can be underrun if the source has a low rate).
+  ;table-slice-size = 100
+
+  ; The table slice type (default|arrow).
```

```suggestion
  ; The table slice type (default|arrow|msgpack).
```

It's okay to include his in here, even if the OSS version doesn't have it. When selecting this, the error message should "this is a pro feature".

---

Let's discuss this in tomorrow's standup, as there are some additional options in commands in VAST Pro that are not mentioned here at all yet.

---

I think it's better not to block this PR from being merged because of this. I'm resolving this without adding `msgpack`—for now.